Swat/SwatTextareaEditor.php

```diff
@@ -262,11 +268,12 @@ protected function getConfig()
         $modes = $this->modes_enabled ? 'yes' : 'no';
         $image_server = $this->image_server ? $this->image_server : '';
 
-        $has_api_key = (self::$tiny_mce_api_key !== null && !empty(self::$tiny_mce_api_key));
+        $has_api_key =
+            self::$tiny_mce_api_key !== null && !empty(self::$tiny_mce_api_key);
         $paste_plugin = $has_api_key ? ' powerpaste' : ' paste';
 
         $config = array(
-            'selector' => '#'.$this->id,
+            'selector' => '#' . $this->id,
```

Comment:
did prettier add this space? Typically we just do `'.$var`

Reply:
prettier did